tests/integration_tests.rs

```diff
@@ -312,6 +313,20 @@ fn can_pass_input_to_command_from_a_file() {
         .stdout(predicate::str::contains("This text is part of a file"));
 }
 
+#[cfg(windows)]
+#[test]
+fn can_pass_input_to_command_from_a_file() {
+    hyperfine()
+        .arg("--runs=1")
+        .arg("--input=example_input_file.txt")
+        .arg("--show-output")
+        .arg("type example_input_file.txt")
```

Comment:
This is not testing the right thing, I believe? The `cat` test above on Linux makes sure that we can pipe input from `example_input_file.txt` into the benchmarked command. But this looks to me like you ignore that and simply output the contents if `example_input_file.txt` using `type`.